.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode9.1
+osx_image: xcode9.3beta
```

Comment:
only beta?

Reply:
Yeap, I was having issues with Travis because I was trying to put xcode9.3, but since it still doesn't have this it was running to the default xcode8.3, breaking it all 😂Took some commits and tests to notice that this was the issue